Parsing_onliner_project/categories.py

```diff
@@ -0,0 +1,75 @@
+"""Storage module for class OnlinerArticle, OnlinerCategory and MainOnlinerPageLinks"""
+from typing import List
+
+from parsing import OnlinerHTMLParser
+from httpRequests import HTTPClient
+
+
+class OnlinerArticle:
+    """Class for gets information about articles"""
+    DEFAULT_HEADERS = {'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64)'
+                                     ' AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36'}
+
+    def __init__(self, url_article):
+        """
+        :param url_article: article url address
+        """
+        self.url = url_article
+        self.onliner_articles = self.__get_articles_info()
+
+    def __get_articles_info(self) -> List[dict]:
+        """
+        Method for gets information from articles
+        :return: list with information about article name, article date and article author for articles
+        """
+        response = HTTPClient.get(self.url, OnlinerArticle.DEFAULT_HEADERS)
+        if response.status_code == 200:
+            articles_info = OnlinerHTMLParser.parser_onliner_articles(response.text)
+            return articles_info
+        print(f'Error with {OnlinerArticle.__get_articles_info.__name__}')
```

Comment:
remove this print.
Tou should log the error message and return empty list instead